packages/common/tests/eips.spec.ts

```diff
@@ -58,4 +58,16 @@ tape('[Common]: Initialization / Chain params', function (t: tape.Test) {
 
     st.end()
   })
+
+  t.test('returns the right block delay for EIP3554', function (st) {
+    for (const fork of ['muirGlacier', 'berlin']) {
+      const c = new Common({ chain: 'mainnet', hardfork: fork })
+      let delay = c.param('pow', 'difficultyBombDelay')
+      st.equal(delay, 9000000)
+      c.setEIPs([3554])
+      delay = c.param('pow', 'difficultyBombDelay')
+      st.equal(delay, 9500000)
+    }
+    st.end()
+  })
```

Comment:
Ah, this test is actually somewhat misplaced - just noticing on rebasing - this test file is just for testing generic EIP switch logic, and not the EIPs themselves (actually all about this library is about "EIPs" 😛 ).

So this would rather go to `params.spec.ts` since this is testing a parameter switch. No drama, but we should move on occasion to not get more blurry here.

Reply:
added this as an item to Tiny Fixes Meta Issue (#905)

Reply:
Ah, I had already updated this along my test work at some point (will remove again from the list). 😋

Reply:
(actually: I did a lot of explicit code changes, updates, "code decisions" on my work on the 1559 PR, so it would be very helpful if you could also take some time short-term and go especially through the commits that I have done on top of Jochem's work (I have already reviewed the most part of this): https://github.com/ethereumjs/ethereumjs-monorepo/pull/1148/commits

This would help us a lot to keep our release schedule and reviewing this huge 1559 PR is like doing 10-20 other "normally-sized" PRs we do on other occasions. 😛  )